packages/devtools_app/lib/src/screens/memory/panes/tracing/tracing_view.dart

```diff
@@ -38,6 +40,21 @@ class TracingPaneState extends State<TracingPane> {
 
   @override
   Widget build(BuildContext context) {
+    if (widget.controller.mode == ControllerCreationMode.offlineData) {
+      return Expanded(
+        child: Markdown(
+          data: '''
+Historical data for tracing is not supported.
+Upvote [the issue](https://github.com/flutter/devtools/issues/7801) if you want the support to be implemented.
+''',
+          styleSheet: MarkdownStyleSheet(
+            p: Theme.of(context).regularTextStyle,
+          ),
+          onTapLink: (text, url, title) =>
+              unawaited(launchUrlWithErrorHandling(url!)),
+        ),
+      );
+    }
```

Comment:
why are we adding this? We should just implement the serialization for any allocation traces that are available in this view (as part of a separate PR).

Reply:
Removed this.